### ISSUE TITLE
runtest.sh failure resolving CoreDisTools on Mac OS X 10.12.4

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/coreclr#10707]
When I run runtest.sh, I see this error:
```
error: Unable to resolve 'runtime.osx.10.12-x64.Microsoft.NETCore.CoreDisTools (>= 1.0.1-prerelease)' for 'DNXCore,Version=v5.0'.
```
(the test run continues nonetheless)

**Comment by BruceForstall** [comment]
@swaroop-sridhar ideas? who should own this?

**Comment by swaroop-sridhar** [comment]
This non-fatal error shows up on Linux and Mac.
I think the correct thing to do is to fix #7861
